composer/loggers/logger_hparams.py

```diff
@@ -202,8 +215,9 @@ class TQDMLoggerHparams(LoggerCallbackHparams):
     """:class:`~composer.loggers.tqdm_logger.TQDMLogger`
     hyperparameters.
 
-    See :class:`~composer.loggers.tqdm_logger.TQDMLogger`
-    for documentation.
+    Args:
+        config (dict or None, optional):
```

Comment:
Config is an argument for `initailize_object` -- the TQDM Logger Hparams has no parameters.